### ISSUE TITLE
lambdaj-1.9-with-dependencies.jar contains 240kb of unnecessary files and classes

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dschulten/lambdaj#8]
```
lambdaj-1.9-with-dependencies.jar contains many unnecessary files and
classes.  The following files and classes are included in the jar and
should be removed:


C:/              (looks to be a copy of ch/lambdaj class files)
junit/           (copy of junit class files)
org/apache       (apache commons logging class files)
org/junit/       (junit class files)

None of the files and classes seem to be needed for lambdaj runtime, 
as best as I can tell, the only files required are:

ch/*
net/*
org/hamcrest/*




```

Original issue reported on code.google.com by `tpoin...@gmail.com` on 23 Jun 2009 at 5:49

**Comment by GoogleCodeExporter** [comment]
```
To second this, some classes seem to have been included accidentally on Windows
during build. Here is a quick snapshot:
C:/
C:/data/
C:/data/development/
C:/data/development/worksapce/
C:/data/development/worksapce/lambdaj/
C:/data/development/worksapce/lambdaj/target/
C:/data/development/worksapce/lambdaj/target/classes/
C:/data/development/worksapce/lambdaj/target/classes/ch/
C:/data/development/worksapce/lambdaj/target/classes/ch/lambdaj/
C:/data/development/worksapce/lambdaj/target/classes/ch/lambdaj/function/
C:/data/development/worksapce/lambdaj/target/classes/ch/lambdaj/function/aggrega
te/
C:/data/development/worksapce/lambdaj/target/classes/ch/lambdaj/function/argumen
t/
C:/data/development/worksapce/lambdaj/target/classes/ch/lambdaj/function/compare
/
C:/data/development/worksapce/lambdaj/target/classes/ch/lambdaj/function/convert
/
C:/data/development/worksapce/lambdaj/target/classes/ch/lambdaj/function/matcher
/
C:/data/development/worksapce/lambdaj/target/classes/ch/lambdaj/group/
C:/data/development/worksapce/lambdaj/target/classes/ch/lambdaj/proxy/
C:/data/development/worksapce/lambdaj/target/classes/ch/lambdaj/util/
C:/data/development/worksapce/lambdaj/target/classes/ch/lambdaj/function/aggrega
te/Aggregator.class
C:/data/development/worksapce/lambdaj/target/classes/ch/lambdaj/function/aggrega
te/Chooser.class

```

Original comment by `celal.zi...@gmail.com` on 25 Jun 2009 at 6:20

**Comment by GoogleCodeExporter** [comment]
Original comment by `mario.fu...@gmail.com` on 26 Jun 2009 at 7:12
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in release 1.10
```

Original comment by `mario.fu...@gmail.com` on 1 Jul 2009 at 10:51
- Changed state: **Fixed**
